rulesets/src/naming.ruleset.yml

```diff
@@ -1,5 +1,47 @@
 rules:
-  # domain references (URN-like values)
+  ##### General #####
+  sps-no-keyword-conflicts:
+    description: Names that may conflict with keywords in common programming languages SHOULD NOT be used.
+    severity: warn
+    given: "$..properties.*~"
+    then:
+      function: pattern
+      functionOptions:
+        notMatch: ^(abstract|continue|for|new|switch|assert|default|goto|package|synchronized|boolean|do|if|private|this|break|double|implements|protected|throw|byte|else|import|public|throws|case|enum|instanceof|return|transient|catch|extends|int|short|try|char|final|interface|static|void|class|finally|long|volatile|const|float|native|super|while)$
+
+  ##### Property Names #####
+  sps-camel-case-properties:
+    description: Property names and acronyms MUST be in camelCase.
+    severity: error
+    formats: [oas3]
+    given: "$..properties.*~"
+    then:
+      function: casing
+      functionOptions:
+        type: camel
+        disallowDigits: true
+
+  sps-disallowed-prepositions:
```

Comment:
We use these names in the "codied name" column on the confluence page to track whether we coded the rule or not. Would you be able to go and add any of the rules without a "codified name" to the [table](https://atlassian.spscommerce.com/wiki/display/DPE/SPS+API+Standards+Codified+Ruleset)?

Reply:
I've updated the confluence page